docs/workforce-management-framework/topics/con-PassportAuth-Module.adoc

```diff
@@ -0,0 +1,38 @@
+[]
+= {WFM-RC-NameShort} Passport Auth
+
+{WFM-RC-NameShort} authentication and authorization module using link:http://passportjs.org[PassportJS].
+This module allows the:
+- Creation and initialization of a Passport authentication service using Passport's local strategy.
+- Protection of express routes from requests by user authentication and authorization.
+- Usage of persistent login sessions using link:https://github.com/expressjs/session[express-session].
+
+
+== Quick Start
+=== Setup
```

Comment:
Setup? I don't see any setup - I only see a snippet of code. Maybe a line or two explaining what is happening below.